docs/getting-started/endgame-sensor-FDA-ven.asciidoc

```diff
@@ -0,0 +1,82 @@
++
+You must also grant Full Disk Access to `com.endgame.systemextension`.
+
+- `13.0` and above: Grant the <<endpoint-endgame-sensor, esensor file>> Full Disk Access.
+
+NOTE: The following instructions apply to the Elastic Endgame sensor only. To see requirements for the {elastic-endpoint}, refer to <<deploy-elastic-endpoint>>.
+
+[discrete]
+[[system-extension-ven]]
+== Approve the system extension for the Elastic Endgame sensor
+
+To fully protect endpoints from malware and other cybersecurity threats when using Elastic Endgame with system extensions, you must enable the system extension during installation on macOS Big Sur (11.0) and later.
+
+When you receive the following prompt to approve loading the system extension:
+image::images/fda/endgame-sys-ext-blocked-ven.png[System extension blocked]
+
+. Open the *System Settings* application.
+. Select *Privacy & Security*.
+. In the right pane, scroll down to the Security section. Click **Allow** to allow the Elastic Endgame system extension to load.
++
+image::fda/endgame-allow-sys-ext-ven.png[]
++
+. Enter your username and password and click **Modify Settings** to save your changes.
+
+[discrete]
+[[allow-network-filter-content-ven]]
+== Approve network content filtering for Elastic Endgame
+
+After successfully loading the Elastic Endgame system extension, an additional message appears, asking to allow Elastic Endgame to filter network content.
+
+[role="screenshot"]
+image::fda/endgame-allow-network-filter-ven.png[]
+
+Click *Allow* to enable content filtering for the Elastic Endgame system extension. Without this approval, Elastic Endgame cannot receive network events and, therefore, cannot enable network-related features such as <<host-isolation-ov, host isolation>>.
+
+[discrete]
+[[endpoint-endgame-sensor-ven]]
+== Enable Full Disk Access for the Elastic Endgame sensor
+
+For the Elastic Endgame sensor to detect events from a macOS host, you must enable Full Disk Access for the `esensor` file, which appears once you've downloaded the sensor on your host.
+
+. Open the *System Settings* application.
+. Select *Privacy & Security*.
+. From the right pane, select *Full Disk Access*.
++
+[role="screenshot"]
+image::install-endpoint-ven/select_fda_ven.png[Select Full Disk Access]
++
+. Click the *+* button to view *Finder*.
+. The system may prompt you to enter your username and password if you haven't already.
++
+[role="screenshot"]
+image::fda/endgame-enter-login-details-to-confirm-ven.png[]
++
+. Navigate to `/Library/Endgame`, then select the `esensor` file.
+. Click *Open*.
+. In the *Privacy* tab, confirm that the `com.endgame` and `esensor` file appears in the list of applications with Full Disk Access permission.
```

Comment:
```suggestion
. In the *Privacy* tab, confirm that the `com.endgame` and `esensor` files appear in the list of applications with Full Disk Access permission.
```

Reply:
On line 22 we referred to `com.endgame.systemextension` – should the filenames match or are `com.endgame` and `com.endgame.systemextension` separate files?

Reply:
I believe they're separate files. @ricardoungureanu would you mind confirming?